controllers/seedgen_controller.go

```diff
@@ -78,6 +78,18 @@ var (
 
 const (
 	EnvSkipRecert = "SEEDGEN_SKIP_RECERT"
+
+	// SeedGen reconciler phases
+	phaseInitial    = "initial"    // SeedGen hasn't started yet
+	phaseGenerating = "generating" // SeedGen is in the first phase of work, ending with the launch of the imager
+	phaseFinalizing = "finalizing" // SeedGen has previously launched the imager and is in the final phase of seed generation
+	phaseCompleted  = "completed"  // SeedGen has successfully completed
+	phaseFailed     = "failed"     // SeedGen has failed
```

Comment:
could make the them typed and include them in the seedgen apis for consistency? e.g https://github.com/openshift-kni/lifecycle-agent/blob/main/api/v1alpha1/types.go#L45-L46

Reply:
I can certainly make it typed, but this is an internal state that won't be exposed in the API file

Reply:
Ah yeah just typed work in that case